jaxtyping/_decorator.py

```diff
@@ -27,8 +27,10 @@
 from collections.abc import Callable
 from typing import (
     Any,
+    ContextManager,
```

Comment:
This is [deprecated in favour of `contextlib.AbstractContextManager`](https://docs.python.org/3/library/typing.html#typing.ContextManager).